scripts/run-django-dev.sh

```diff
@@ -5,7 +5,7 @@
 health_urls=(
     "http://watch:8052/health"
 )
-wait_time=300
+wait_time=600
```

Comment:
If you're running in prod mode and node_modules is empty, 5 minutes was not enough time for me to install and build.

I'm not sure what a good timeout would be. If this does timeout, you can always just restart the `web` container.